server/index.js

```diff
@@ -10,13 +10,18 @@ app.use(express.static(path.join(__dirname, '/../client/dist')));
 app.use(bodyparser.json());
 app.use(bodyparser.urlencoded({ extended: true }));
```

Comment:
You should not need bodyparser here - there are no `POST` endpoints that read a request body.